docs/core/tools/csproj.md

```diff
@@ -223,6 +223,24 @@ The following example specifies the fallbacks only for the `netcoreapp2.1` targe
 </PackageTargetFallback >
 ```
 
+## Build events
+
+The way that pre-build and post-build events are specified in the project file has changed. The properties PreBuildEvent and PostBuildEvent are not recommended in the SDK-style project format, because macros such as $(ProjectDir) are not resolved. For example, the following code is no longer supported:
+
+```xml
+<PropertyGroup>
+    <PreBuildEvent>"$(ProjectDir)PreBuildEvent.bat" "$(ProjectDir)..\" "$(ProjectDir)" "$(TargetDir)" />
+</PropertyGroup>
+```
+
+In SDK-style projects, use an MSBuild target named `PreBuild` or `PostBuild`. For the preceding example, use the following code:
```

Comment:
Aren't the names `PreBuild` and `PostBuild` arbitrary? Really the requirement is to create a new target that calls `BeforeTargets="PreBuildEvent"` and `AfterTargets="PostBuildEvent"`

Yes?

Reply:
You're probably right, let me confirm and update as necessary.

Reply:
What I found is that if you use different names for the targets, they do run as expected.  However, the VS IDE does not find and detect them and show them in the Build Events tab of the Project Designer, where they can be edited.  I could add a note that explains that, maybe.

Reply:
Interesting! I think that it's OK as it is, but maybe just add another sentence explaining that each should be setting the Before/After targets property.